### PR TITLE
Age/Sex/Race and Arson summary tables

### PR DESCRIPTION
**Pull request by catherinedevlin**
Creates new /arson, /arrests/age_sex, /arrests/race, and /arrests/ethnicity endpoints.  They are summarized data somewhat like the /incidents/count/ RETA data is, so they piggyback off that code.  Unfortunately each of them is a standalone data set incompatible with the RETA records.  :(

### CODE REVIEW

File: crime_data/resources/arrests.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    tables = cdemodels.ArrestsByEthnicityTableFamily()
+
+
+class ArrestsCountByAgeSex(ArrestsCountResource):
+
+    tables = cdemodels.ArrestsByAgeSexTableFamily()
+
+
+"""
+Big (?) problem here: whole different set of classification codes
```
Comment (harrisj) [inline crime_data/resources/arrests.py]: Ooof, I wonder if we'll have to create meta-mappings from a third neutral set to each of these.

File: crime_data/resources/incidents.py [hunk 2]
```diff
@@ -19,31 +14,17 @@ class IncidentsList(CdeResource):
     schema = marshmallow_schemas.NibrsIncidentSchema(many=True)
     tables = cdemodels.IncidentTableFamily()
 
-    @use_args(ArgumentsSchema)
+    @use_args(marshmallow_schemas.ArgumentsSchema)
```
Comment (harrisj) [inline crime_data/resources/incidents.py]: Just for my own education, is there a specific reason you decided to explicitly scope the ArgumentsSchema here

File: tests/functional/test_arson.py [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+
+class TestTuningPage:
+    def test_tuning_page_exists(self, user, testapp):
+        res = testapp.get('/arson/?tuning=1')
+        assert res.status_code == 200
+        assert b'<!DOCTYPE html>' in res.body
+
+
+class TestArsonCountEndpoint:
```
Comment (harrisj) [inline tests/functional/test_arson.py]: Yay!!!

**Comment by harrisj** [comment]
Still needs a rebase, but I will handle it

**Comment by harrisj** [comment]
Okay, I've tried rebasing but it seems to have gone terribly wrong

**Comment by cacraig** [comment]
No problem @harrisj , I'll see if I can get this sorted out. Thanks!

**Comment by harrisj** [comment]
I appreciate your efforts here. Merging

**Pull request merged**